iModelCore/ECDb/ECDb/SchemaManagerDispatcher.cpp

```diff
@@ -1002,18 +1002,24 @@ void DumpSchemasToFile(BeFileName const& parentDirectory, bvector<ECSchemaCP> co
 //+---------------+---------------+---------------+---------------+---------------+------
 BentleyStatus MainSchemaManager::ImportSchemas(SchemaImportContext& ctx, bvector<ECSchemaCP> const& schemas, SchemaImportToken const* schemaImportToken) const
     {
-    #ifndef NDEBUG
+    //#define ALLOW_ECDB_SCHEMAIMPORT_DUMP
+    #if !defined(NDEBUG) && defined(ALLOW_ECDB_SCHEMAIMPORT_DUMP)

```

Comment:
```suggestion
    #if defined(ALLOW_ECDB_SCHEMAIMPORT_DUMP)
```
Now you can enable this feature without having to deal with slower debug performance and annoying assertions.

Reply:
Yeah, it's better this way, let's do as Paul suggests.